.travis.yml

```diff
@@ -6,11 +6,11 @@
   matrix:
     include:
       - os: linux
-        go: 1.7
+        go: 1.7.x
```

Comment:
Even if Go releases keep backwards compatibility, things can still break. In my opinion it's better to do that manually.

In any case, it seems like this is the strategy this tool uses. I don't want to change it now.